debian-snap-base/Dockerfile.tmpl

```diff
@@ -1,12 +1,11 @@
-FROM unocha/debian-base-s6:%%UPSTREAM%%
+FROM unocha/nodejs-base:%%UPSTREAM%%
```

Comment:
This is based on Alpine isn't it?  We use the debian base image for snap because we don't know that Chrome works reliably on musl systems.

Reply:
I generated `unocha/nodejs-base` images for both alpine and debian. The ones with alpine are named `12-alpine` and the ones with debian `12-buster`.
Happy to change `buster` to `debian` when we get things rolling.